charts/pega/values-large.yaml

```diff
@@ -15,8 +15,8 @@ global:
   actions:
     execute: "deploy"
   # Add custom certificates to be mounted to container
-  #to support external secrets
-  #supports multiple externalSecrets , add externalSecret names as array of strings
+  # to support external secrets
+  # supports multiple externalSecrets , add externalSecret names as array of strings
```

Comment:
update these across the 3 values.yaml files>
 to support an external secret, specify your secret using the `certificates` parameter.
 to support multiple external secrets, specify each of your external secrets as an array of comma-separated strings using the `certificatesSecrets` parameter.

Reply:
here we use certificates parameter to provide list of certificates as plain text.
Where as certificatesSecrets takes names of external secrets as array of strings.

Updated accordingly in the latest commit , please review